src/routes/Onboarding/FutureWork.tsx

```diff
@@ -12,7 +12,7 @@ export default function FutureWork() {
 
   useEffect(() => {
     // We do want to update both the state and editor here.
-    codeManager.updateCodeStateEditor(bracket)
```

Comment:
wait so im confused you dont want to update the state for these? if that state is now never used we can nukke it

Reply:
well this isn't my code so not super sure, but there were onboarding tests that failed before this change that are now passing so I think this is what we want.

Reply:
updateCodeEditor is still used in a couple of more places so I'll keep it for now.